layers/+tools/shell/funcs.el

```diff
@@ -27,6 +27,15 @@
   (let ((default-directory (projectile-project-root)))
     (call-interactively 'spacemacs/default-pop-shell)))
 
+(defun spacemacs/projectile-run-default-shell ()
+  "Run the default shell in the project root directory."
+  (interactive)
+  (when-let ((shell (cl-case shell-default-shell
+                 ('multi-term #'projectile-multi-term-in-root)
+                 ('ansi-term nil)
```

Comment:
on second thought. maybe we can do this
```elisp
(call-interactively (if (eq shell-default-shell 'multi-term)
                        #'projectile-multi-term-in-root
                      (or (intern-soft (format "projectile-run-%s") shell-default-shell)
                          #'projectile-run-shell)))
```

And change the docstring accordingly.